tests/integration/openconfig-bgp/testdata/tc040.json

```diff
@@ -144,7 +144,7 @@
                         "route-flap-damping": false, 
                         "enabled": false, 
                         "peer-as": 5400, 
-                        "peer-group": false, 
+                        "peer-group": 0, 
```

Comment:
This doesn't look valid to me -- since `peer_group` is a leafref, then this should either be a string or nil?